Riot/Assets/en.lproj/Vector.strings

```diff
@@ -2430,18 +2430,21 @@ To enable access, tap Settings> Location and select Always";
 "user_session_learn_more" = "Learn more";
 "user_session_verified_additional_info" = "Your current session is ready for secure messaging.";
 "user_session_unverified_additional_info" = "Verify your current session for enhanced secure messaging.";
+"user_session_verification_unknown_additional_info" = "Verify your current session to reveal this session's verification status.";
 
 "user_session_push_notifications" = "Push notifications";
 "user_session_push_notifications_message" = "When turned on, this session will receive push notifications.";
 
 "user_other_session_security_recommendation_title" = "Security recommendation";
 "user_other_session_unverified_sessions_header_subtitle" = "Verify your sessions for enhanced secure messaging or sign out from those you don’t recognize or use anymore.";
-"user_other_session_unverified_current_session_details" =  "%@ · Your current session";
+"user_other_session_current_session_details" =  "Your current session";
 
 // First item is client name and second item is session display name
 "user_session_name" = "%@: %@";
 
-"user_session_item_details" = "%@ · Last activity %@";
+/* %1$@ will be the verification state and %2$@ will be user_session_item_details_verification_unknown or user_other_session_unverified_current_session_details */
+"user_session_item_details" = "%1$@ · %2$@";
```

Comment:
This change of content seems minor if there are a few translated strings that aren't updated right away so I didn't create a new key for it.